core/src/main/resources/defaults/clustertemplates/azure/azure-dataengineering-ha-702.json

```diff
@@ -0,0 +1,174 @@
+      {
+        "nodeCount": 3,
+        "name": "quorum",
+        "type": "CORE",
+        "azure": {},
+        "recoveryMode": "MANUAL",
+        "template": {
+          "azure": {
+            "encrypted": false,
+            "managedDisk": true
+          },
+          "instanceType": "Standard_D8_v3",
+          "rootVolume": {
+            "size": 50
+          },
+          "attachedVolumes": [
+            {
+              "size": 100,
+              "count": 1,
+              "type": "StandardSSD_LRS"
+            }
+          ],
+          "cloudPlatform": "AZURE"
+        },
+        "recipeNames": [],
+        "cloudPlatform": "AZURE"
+      },
+      {
+        "nodeCount": 1,
+        "name": "manager",
+        "type": "GATEWAY",
+        "azure": {},
+        "recoveryMode": "MANUAL",
+        "template": {
+          "azure": {
+            "encrypted": false,
+            "managedDisk": true
+          },
+          "instanceType": "Standard_D8_v3",
+          "rootVolume": {
+            "size": 50
+          },
+          "attachedVolumes": [
+            {
+              "size": 100,
+              "count": 1,
+              "type": "StandardSSD_LRS"
+            }
+          ],
+          "cloudPlatform": "AZURE"
+        },
+        "recipeNames": [],
+        "cloudPlatform": "AZURE"
+      },
+      {
+        "nodeCount": 0,
+        "name": "compute",
+        "type": "CORE",
+        "azure": {},
+        "recoveryMode": "MANUAL",
+        "template": {
+          "azure": {
+            "encrypted": false,
+            "managedDisk": true
+          },
+          "instanceType": "Standard_D8_v3",
+          "rootVolume": {
+            "size": 50
+          },
+          "attachedVolumes": [
+            {
+              "size": 100,
+              "count": 1,
+              "type": "StandardSSD_LRS"
+            }
+          ],
+          "cloudPlatform": "AZURE"
+        },
+        "recipeNames": [],
+        "cloudPlatform": "AZURE"
+      },
+      {
+        "nodeCount": 3,
+        "name": "worker",
+        "type": "CORE",
+        "azure": {},
+        "recoveryMode": "MANUAL",
+        "template": {
+          "azure": {
+            "encrypted": false,
+            "managedDisk": true
+          },
+          "instanceType": "Standard_D8_v3",
+          "rootVolume": {
+            "size": 50
+          },
+          "attachedVolumes": [
+            {
+              "size": 100,
+              "count": 1,
+              "type": "StandardSSD_LRS"
+            }
+          ],
+          "cloudPlatform": "AZURE"
+        },
+        "recipeNames": [],
+        "cloudPlatform": "AZURE"
+      },
+      {
+        "nodeCount": 2,
+        "name": "master",
+        "type": "CORE",
+        "azure": {},
+        "recoveryMode": "MANUAL",
+        "template": {
+          "azure": {
+            "encrypted": false,
+            "managedDisk": true
+          },
+          "instanceType": "Standard_D16_v3",
```

Comment:
This stood out. Azure masters get 16cores/64GB. AWS gets 8 cores/32GB. Intentional?

Reply:
Good catch. I was following the existing DE Azure definition. Can fix in all the DE templates for Azure.

Reply:
This jira answers many questions https://jira.cloudera.com/browse/CB-4012. This is where the master instance type was set to Standard_D16_v3 CC: @doktoric

Reply:
At the same time opdb master is moving to D8 https://jira.cloudera.com/browse/CB-5275

Reply:
Moved to Standard_D8_v3 for all the DE template masters.